examples/step-66/doc/intro.dox

```diff
@@ -0,0 +1,266 @@
+<br>
+
+<i>
+This program was contributed by Fabian Castelli.
+
+A version of this code was presented and discussed in:
+G.F. Castelli: Numerical Investigation of Cahn-Hilliard-Type Phase-Field
+Models for Battery Active Particles, PhD thesis, Karlsruhe Institute of
+Technology (KIT), 2021. (To be published)
```

Comment:
Can you add this publication to both https://github.com/dealii/publication-list and `doc/doxygen/references.bib` once it's published? You can add a bullet point to the issue I proposed.

Reply:
Sure this comes on the issue list.